tools/python/boutiques/tests/test_import.py

```diff
@@ -4,17 +4,25 @@
 from boutiques import bosh
 from boutiques import __file__ as bfile
 from jsonschema.exceptions import ValidationError
-import os.path as op
 import json
 import os
+import os.path as op
+from os.path import join as opj
 
 
 class TestImport(TestCase):
 
     def test_import_bids_good(self):
         bids_app = op.join(op.split(bfile)[0],
```

Comment:
Since you're using `opj` many places in this file, might as well use it everywhere we use `op.join`? Can't remove the `import os.path as op` anyways since we use other functions of it, but consistency is nice :)